runtime/lua/man.lua

```diff
@@ -1,3 +1,5 @@
+require('vim.compat')
```

Comment:
Not sure if we want to do this implicitly instead. But let's discuss that later.